### PR TITLE
Sign Up Page

### PR DESCRIPTION
**Pull request by m26unkwn**
Feature
1. First Name
2. Last Name
3. Email 
4. Password

### CODE REVIEW

File: styles/auth.css [hunk 1]
```diff
@@ -7,7 +7,7 @@
 }
 
 .auth-head {
-  margin: 2rem 0;
+  margin: 2rem 0 1rem 0;
 }
```
Comment (j24m) [inline styles/auth.css:11]: please provide with consistent indentation:)

**Review by j24m — APPROVED** [review]
Suggested one change, rest of the code seems good, will have to give you 5 stars for a small PR
 : )

File: styles/auth.css (hunk 1 above)
Comment (m26unkwn) [reply styles/auth.css:11]: Thanks @j24m  for pointing out this. Will resolve it soon.

File: screens/signup.html [hunk 2]
```diff
@@ -0,0 +1,173 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Devstore | devlopers mall</title>
+    <link rel="stylesheet" href="../styles/gobal.css" />
+    <link rel="stylesheet" href="../styles/home.css" />
+    <link rel="stylesheet" href="../styles/auth.css" />
```
Comment (dipen2000) [inline screens/signup.html:10]: Have one file which you apply to every page and then import these files to that file so there won't be so much clutter in your HTML files.

File: screens/signup.html [hunk 3]
```diff
@@ -0,0 +1,173 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Devstore | devlopers mall</title>
+    <link rel="stylesheet" href="../styles/gobal.css" />
```
Comment (dipen2000) [inline screens/signup.html:8]: and is this a typo in gobal? It should be global if that is not typo.
Use extension for spell check in VS code so you won't make typos.

**Review by dipen2000 — APPROVED** [review]
Look at the few suggestions I mentioned and other than that it looks good to go

File: screens/signup.html [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+    <link rel="stylesheet" href="../styles/home.css" />
+    <link rel="stylesheet" href="../styles/auth.css" />
+  </head>
+  <body>
+    <main class="main-wrapper">
```
Comment (ashishkatiyar2404) [inline screens/signup.html:13]: this is the header section na. you can use header tag instead of main.

**Review by ashishkatiyar2404 — APPROVED** [review]
suggested one thing rest looks good to me💯
BTW your home page looks awesome🔥

File: screens/signup.html (hunk 3 above)
Comment (m26unkwn) [reply screens/signup.html:8]: Thanks for pointing out this gobal soon going to global. xD

File: screens/signup.html (hunk 2 above)
Comment (m26unkwn) [reply screens/signup.html:10]: Thanks @dipen2000  will look over this suggestions.

File: screens/signup.html (hunk 4 above)
Comment (m26unkwn) [reply screens/signup.html:13]: Its not header @ashishkatiyar2404 its a main wrapper inside there is header which i am going to change after your suggestion to header thanks.

File: screens/login.html [hunk 5]
```diff
@@ -151,7 +153,7 @@ <h3 class="head-title">Login</h3>
             <div class="pass-forget">
               <a href="#">Forget Password?</a>
```
Comment (ShrutiLakhani) [inline screens/login.html:154]: Should be Forgot Password. Also, red/pink color doesn't seem apt here. But,yeah its a personal choice.
Everything else looks good

**Review by ShrutiLakhani — APPROVED** [review]
LGTM!

**Comment by PuneetP16** [comment]
SignUp Page
image

On the Sign up and Sign in page your header/ navigation bar items are looking inconsistent with landing page. Maybe update it as you have kept in home page or add gap between items.

File: styles/home.css [hunk 6]
```diff
@@ -186,9 +186,7 @@
   .logotext {
     display: none;
```
Comment (PuneetP16) [inline styles/home.css:187]: Just a suggestion, for consistent naming of classes, you could separate words using - or _ would look more consistent and easy to distinguish like its an image or text element.

**Review by PuneetP16 — APPROVED** [review]
Looks good to merge.

File: styles/home.css (hunk 6 above)
Comment (m26unkwn) [reply styles/home.css:187]: I'll look upon your suggestions Thanks.

Reason behind changing the header on authentication page.

1. User is already on the authentication page so there is no need to put the login and signup button over there.
2. I am doing flex-center so there will be no more white spaces in header elements.

Comment (PuneetP16) [reply styles/home.css:187]: So you have done it intentionally, that's ok. Btw beautiful landing page design.

**Pull request merged**